app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, file_in, file_out = command.split()
+    if not cmd == "cp":
```

Comment:
It will be better to do it like:
```suggestion
    if cmd == "cp":
        ...
```
There will be no need to use additional return then